Makefile.toml

```diff
@@ -146,6 +146,20 @@ script = '''
 exec --fail-on-error cargo test --package golem-templates --test integration -- --nocapture --test-threads=1 --report-time %{JUNIT_OPTS}
 '''
 
+## ** Single executable **
```

Comment:
i'm a bit unsure about these tasks, because:
 - we have install tasks for installing the debug versions globally, and it also _reuses_ the actual target dir, to make it faster
 - i'm not sure why it depends on set version, i think that should be only used for pubish
 - the run task has wrong description
 
i think the install task should cover the usual dev needs, and having a specialized cargo make command not really helps for many cases (except for booting the server itself), given the golem command most of the time needs an app folder anyway. so i'm not against having such commands, but that could be a simple 'cargo run'

Reply:
Good point regarding cargo run, forgot it exists when adding it.
Feel free to nuke in your next pr